src/session/session_compact.c

```diff
@@ -175,6 +175,22 @@ __compact_handle_append(WT_SESSION_IMPL *session, const char *cfg[])
     return (0);
 }
 
+/*
+ * __compact_timing_stress --
+ *     Optionally add a delay to the compact for debug purposes.
+ */
+static void
+__compact_timing_stress(WT_SESSION_IMPL *session, uint64_t flag, struct timespec *tsp)
+{
+    WT_CONNECTION_IMPL *conn;
+
+    conn = S2C(session);
+
+    // We only want to sleep if the flag is set.
```

Comment:
Fix the C++ style comment.

Reply:
Fixed in 28562ccdda07fcdc52f0b92ce38b9138ad5870dc